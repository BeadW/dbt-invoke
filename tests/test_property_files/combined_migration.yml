version: 2
models:
- name: customers
  description: ''
  columns:
  - name: customer_id
    description: ''
  - name: created_at
    description: ''
- name: orders
  description: ''
  columns:
  - name: order_id
    description: ''
  - name: customer_id
    description: ''
  - name: item_id
    description: ''
  - name: quantity
    description: ''
  - name: order_at
    description: ''
seeds:
- name: items
  description: ''
  columns:
  - name: item_id
    description: ''
  - name: price
    description: ''
  - name: updated_at
    description: ''
snapshots:
- name: items_snapshot
  description: ''
  columns:
  - name: item_id
    description: ''
  - name: price
    description: ''
  - name: updated_at
    description: ''
  - name: dbt_valid_from
    description: ''
  - name: dbt_valid_to
    description: ''
analyses:
- name: revenue_by_daily_cohort
  description: ''
  columns:
  - name: created_on
    description: ''
  - name: total_revenue
    description: ''
