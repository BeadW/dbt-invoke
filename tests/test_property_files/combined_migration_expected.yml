version: 2
models: []
seeds: []
snapshots: []
analyses: []
